multiversx_sdk/multisig/multisig_transactions_factory.py

```diff
@@ -0,0 +1,543 @@
+            contract=contract,
+            function="discardAction",
+            gas_limit=gas_limit,
+            arguments=[U32Value(action_id)],
+        )
+
+    def create_transaction_for_discard_batch(
+        self,
+        sender: Address,
+        contract: Address,
+        action_ids: list[int],
+        gas_limit: int,
+    ) -> Transaction:
+        return self._sc_factory.create_transaction_for_execute(
+            sender=sender,
+            contract=contract,
+            function="discardBatch",
+            gas_limit=gas_limit,
+            arguments=[VariadicValues(items=[U32Value(id) for id in action_ids])],
+        )
+
+    def create_transaction_for_propose_add_board_member(
+        self,
+        sender: Address,
+        contract: Address,
+        board_member: Address,
+        gas_limit: int,
+    ) -> Transaction:
+        return self._sc_factory.create_transaction_for_execute(
+            sender=sender,
+            contract=contract,
+            function="proposeAddBoardMember",
+            gas_limit=gas_limit,
+            arguments=[AddressValue.new_from_address(board_member)],
+        )
+
+    def create_transaction_for_propose_add_proposer(
+        self,
+        sender: Address,
+        contract: Address,
+        proposer: Address,
+        gas_limit: int,
+    ) -> Transaction:
+        return self._sc_factory.create_transaction_for_execute(
+            sender=sender,
+            contract=contract,
+            function="proposeAddProposer",
+            gas_limit=gas_limit,
+            arguments=[AddressValue.new_from_address(proposer)],
+        )
+
+    def create_transaction_for_propose_remove_user(
+        self,
+        sender: Address,
+        contract: Address,
+        user: Address,
+        gas_limit: int,
+    ) -> Transaction:
+        return self._sc_factory.create_transaction_for_execute(
+            sender=sender,
+            contract=contract,
+            function="proposeRemoveUser",
+            gas_limit=gas_limit,
+            arguments=[AddressValue.new_from_address(user)],
+        )
+
+    def create_transaction_for_propose_change_quorum(
+        self,
+        sender: Address,
+        contract: Address,
+        quorum: int,
+        gas_limit: int,
+    ) -> Transaction:
+        return self._sc_factory.create_transaction_for_execute(
+            sender=sender,
+            contract=contract,
+            function="proposeChangeQuorum",
+            gas_limit=gas_limit,
+            arguments=[U32Value(quorum)],
+        )
+
+    def create_transaction_for_propose_transfer_execute(
+        self,
+        sender: Address,
+        contract: Address,
+        receiver: Address,
+        native_token_amount: int,
+        gas_limit: int,
+        opt_gas_limit: Optional[int] = None,
+        abi: Optional[Abi] = None,
+        function: Optional[str] = None,
+        arguments: Optional[list[Any]] = None,
+    ) -> Transaction:
+        if not function:
```

Comment:
Slightly different from JS (regarding the named constructors of `ProposeTransferExecuteContractInput`) - if possible, we should have similar implementations.

https://github.com/multiversx/mx-sdk-js-core/blob/TOOL-329-add-mustisig-factory/src/multisig/multisigTransactionsFactory.ts#L137

Also - naming `ProposeTransferExecuteContractInput` (JS) vs. `ProposeTransferExecuteInput` (PY).

Though, which one to choose?